integration/setup/common.go

```diff
@@ -70,6 +70,17 @@ func common(ctx context.Context, config Config) error {
 		}
 	}
 
+	{
+		config.Logger.LogCtx(ctx, "level", "debug", "message", "ensuring drainerconfig CRD exists")
```

Comment:
The drainerconfig CRD was being created by node-controller, but that got removed as part of the CRD graduation effort.